staubli_rx160_support/tests/roslaunch_test.xml

```diff
@@ -6,4 +7,12 @@
   <group ns="test_rx160__">
     <include file="$(find staubli_rx160_support)/launch/test_rx160.launch"/>
   </group>
+
+  <group ns="load_rx160l__">
```

Comment:
Has been the convention ever since we started roslaunch testing launch files. The underscores were added to avoid any potential clashes with named entities defined in any of the launchfiles-under-test. The formatting of the namespace names is also somewhat inspired on how header include guards are formatted in C/C++.